### PR TITLE
feat(booking-platform): add readme

### PR DESCRIPTION
**Pull request by AinaVendrell**
## Description
Add readme for the example of booking-platform

## Testing

The pull request...

- has unit tests
- has integration tests
- doesn't need tests because... **[provide a description]**

### CODE REVIEW

File: example-hotel-reservation/README.md [hunk 1]
```diff
@@ -1,3 +1,554 @@
...
+
+In this section, we will see the most relevant webchat components that have been used to create this example.
+
+
+#### 1. Cover Component
+
+This is the first component that will appear when a user opens the webchat. It's implemented in `src/webchat/cover-component.js`.
+
+We have used  `styled-components` that allows us to define new React component with our styles attached to it.
+
```
Comment (vanbasten17) [inline example-hotel-reservation/README.md:38]: i would add here the name of the file just before the code snippet (for all code snippets). between **** like here https://github.com/hubtype/botonic-examples/tree/master/example-nlu#22-creating-your-own-custom-model

File: example-hotel-reservation/README.md [hunk 2]
```diff
@@ -1,3 +1,554 @@
...
+We have used a CustomMessage to create the **Hotel Form** that appears after the user chooses the hotel to book.
+
+In this component, we have also used `styled-components` and `material-ui` to create the **TextField**, the **Autocomplete** and the **DatePicker** components.
+
+The structure is very similar to the **CustomCover** component, but in this case we need to export the component as:
+
+```javascript
+export default customMessage({
+  name: 'hotel-form',
+  component: HoltelForm,
```
Comment (vanbasten17) [inline example-hotel-reservation/README.md]: HoltelForm --> HotelForm, change also the source code

File: example-hotel-reservation/README.md [hunk 3]
```diff
@@ -1,3 +1,554 @@
...
+          the reservation, we would need some more information.
+        </Text>
+        <HotelForm hotel={this.props.hotel} />
+      </>
+    )
+  }
+}
+```
+
+ <br/>
```
Comment (vanbasten17) [inline example-hotel-reservation/README.md]: is this `<br/>` necessary?

File: example-hotel-reservation/README.md [hunk 4]
```diff
@@ -1,3 +1,554 @@
...
+export default customMessage({
+  name: 'rate-user-message',
+  component: RateUserMessage,
+  defaultProps: {
+    from: 'user',
+  },
+})
+```
+
+<br/>
```
Comment (vanbasten17) [inline example-hotel-reservation/README.md]: <br/> ?

File: example-hotel-reservation/README.md [hunk 5]
```diff
@@ -1,3 +1,554 @@
...
+      ],
+      menu: {
+        darkBackground: true,
+        custom: CustomPersistentMenu,
+      },
+    },
+  },
+}
+```
+In this case, we have created a `persistentMenu` with three buttons the first one will open a webview, the second one will send the payload 'carousel' and the last one will close the menu.
```
Comment (vanbasten17) [inline example-hotel-reservation/README.md]: with three buttons --> with three buttons:

**Review by ericmarcos — CHANGES_REQUESTED** [review]
Very nice!! But I miss links to our documentation. For example, when you talk about custom messages, put a link to our docs article about this topic, so users can go there easily and learn more details if they want. Same with persistent menu, etc.

The goal of this readmes is not to explain all the basic concepts, but to show what is possible, how this example accomplish some goals (code) and point to the right place in the docs to learn more.

File: example-hotel-reservation/README.md [hunk 6]
```diff
@@ -1,3 +1,554 @@
...
+  })
+}
+```
+The last step, is to add the new webview in `src/webviews/index.js` like:
+
+```javascript
+import WebviewReserva from './components/check-reservations'
+
+export const webviews = [WebviewReserva]
+```
```
Comment (vanbasten17) [inline example-hotel-reservation/README.md:569]: add a closing message like: ...and we are done 🎉

**Review by vanbasten17 — APPROVED** [review]
Nice job!! 👏 👏 👏

**Comment by AinaVendrell** [comment]
requested changes applied

File: example-hotel-reservation/README.md [hunk 7]
```diff
@@ -1,3 +1,571 @@
...
+          <em>
+            We will not store the fulfilled information. You can fake the data.
+          </em>
+        </p>
+      </Container>
+    )
+  }
+}
+```
+We have created some functions to check and handle the user data. Moreover, to store the relevant information in the session we have used the `updateUser()` function.
```
Comment (elozano98) [inline example-hotel-reservation/README.md:193]: function or method?

File: example-hotel-reservation/README.md [hunk 8]
```diff
@@ -1,3 +1,571 @@
...
+
+Finally, to close the component, we have called the `closeComponent()` function that can be found in the props. In this case, we also used `sendText` to add a message form the user after the component is closed. That then will be captured in the routes and will execute the next action.
```
Comment (elozano98) [inline example-hotel-reservation/README.md:195]: function or method?

File: example-hotel-reservation/README.md (hunk 7 above)
Comment (vanbasten17) [reply example-hotel-reservation/README.md:193]: function

File: example-hotel-reservation/src/webchat/hotel-form-message.js [hunk 9]
```diff
@@ -27,7 +27,7 @@ const Button = styled.button`
   color: white;
 `
 
-class HoltelForm extends React.Component {
```
Comment (elozano98) [inline example-hotel-reservation/src/webchat/hotel-form-message.js:30]: fighting against dyslexia :joy:

File: example-hotel-reservation/README.md (hunk 8 above)
Comment (vanbasten17) [reply example-hotel-reservation/README.md:195]: function

**Review by elozano98 — APPROVED** [review]
Really good job with the README! :+1:

File: example-hotel-reservation/src/webchat/hotel-form-message.js (hunk 9 above)
Comment (AinaVendrell) [reply example-hotel-reservation/src/webchat/hotel-form-message.js:30]: hahaha molt heavy

**Pull request merged**